includes/functions/llms.functions.certificate.php

```diff
@@ -107,7 +107,7 @@ function llms_get_certificate_content( $id = 0 ) {
  * }
  */
 function llms_get_certificate_fonts() {
```

Comment:
function needs a new `@since [version] description...` changelog added to it

and the file's `@version` tag should be replaced with the `[version]` placeholder

Reply:
Added!